projects/plugins/jetpack/extensions/blocks/story/story.php

```diff
@@ -186,13 +212,20 @@ function render_video( $media ) {
 	}
 
 	if ( ! empty( $poster_url ) ) {
+		$poster_width  = esc_attr( $meta_width );
+		$poster_height = esc_attr( $meta_height );
+		$content_width = Jetpack::get_content_width();
```

Comment:
I played with using `EMBED_SIZE` here instead since the `content_width` value is generally larger than we need for the story preview. However WP.com seems to strip the value from the static fallback content and apply resize parameters based on `Jetpack::get_content_width()` anyway, which will then be different from the value in the video element's `poster` attributed, causing the poster image to be downloaded twice at two different sizes.

Reply:
I wonder if we should sanitize (or rather cast as int) `Jetpack::get_content_width()` too, since it can be impacted by values set in themes.

Reply:
Ah that's interesting - sounds like a good precaution, added in a250d87.